public/javascripts/script.js

```diff
@@ -8,52 +8,51 @@ document.addEventListener('DOMContentLoaded', () => {
 
 //************START GOOGLE MAPS CODE************
 
-//Object of locations
-var locations={
-  building:{
-    center: {
-      lat: 25.766675,
-      lng: -80.196130  
-    }
-  }
-  }
 
-// Initialize and add the map
-function initMap() {
+var map, infoWindow;
+var pos = {};
 
-  var map = new google.maps.Map(
-      document.getElementById('map'), {
-          zoom: 16,
-          center: {lat: 25.766675, lng:-80.196130}
-      });
-
-        // Construct the circle for each value in citymap.
-        // Note: We scale the area of the circle based on the population.
-        for (var city in locations) {
-          // Add the circle for this city to the map.
-          var cityCircle = new google.maps.Circle({
-            strokeColor: '#FF0000',
-            strokeOpacity: 0.8,
-            strokeWeight: 2,
-            fillColor: '#FF0000',
-            fillOpacity: 0.35,
-            map: map,
-            center: {lat: 25.766675, lng:-80.196130},
-            radius: 80 //Math.sqrt(citymap[city].population) * 100
-              
-          });
-        }
-
-  // The marker, positioned at park
-  var marker = new google.maps.Marker({
-      position: building,
-      map: map
+function initMap() {
+  map = new google.maps.Map(document.getElementById('map'), {
+    center: { lat: pos.lat, lng: pos.lng },
+    zoom: 13
   });
-  //To make multiple markers
-  // var marker2 = new google.maps.Marker({
-  //     position: lake,
-  //     map: map
-  // });
-
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(function (position) {
+      pos.lat = position.coords.latitude,
```

Comment:
Might be smart to move obtaining the position + returning it in its own helper function.  Currently this function is very multi-purpose and violates SRP.